buildlib/pr/main.yml

```diff
@@ -1,12 +1,16 @@
 variables:
   DOCKER_OPT_VOLUMES: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
-  DOCKER_OPT_GPU: --gpus all --ulimit memlock=-1:-1 --device=/dev/infiniband/
+  DOCKER_OPT_IB: --ulimit memlock=-1:-1 --device=/dev/infiniband/
+  DOCKER_OPT_GPU: --gpus all $(DOCKER_OPT_IB)
```

Comment:
i think it should be called DOCKER_OPT_GPU

Reply:
sorry?